resident/resident-service/src/main/java/io/mosip/resident/constant/PacketStatus.java

```diff
@@ -37,7 +37,11 @@ public static Optional<String> getStatusCode(String statusCode, Environment env)
 	}
 
 	private static List<String> getStatusCodeList(PacketStatus packetStatus, Environment env) {
-		return List.of(env.getProperty(packetStatus.statusCodePropertyName).split(","));
+		List<String> statusCode = null;
```

Comment:
Initialize it with empty list. (List.of())

Reply:
Updated as per review comment